src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -545,11 +548,7 @@ def ConnectBLE(self, discriminator, setupPinCode, nodeid) -> PyChipError:
                     self.devCtrl, discriminator, setupPinCode, nodeid)
             ).raise_on_error()
 
-            # TODO: Change return None. Only returning on success is not useful.
-            # but that is what the previous implementation did.
-            res = self._commissioning_complete_future.result()
-            res.raise_on_error()
-            return res
+            return self._commissioning_complete_future.result()
```

Comment:
Function marked as return None, but your are returning something here.

Reply:
I do set the result of `_commissioning_complete_future` to `None`, so this technically should be fine. I was considering both directions, I felt this will make it easier to change the return value.

But it has the risk of accidental return value change. And given you would not expect a return statement here, it is probably better to get rid of it :sweat_smile: